publications-2019.bib

```diff
@@ -1,5 +1,222 @@
 % Encoding: US-ASCII
 
+@article{heinlein2019applying,
+  title={Applying the FROSch Overlapping Schwarz Preconditioner for Dislocation Mechanics in Deal. II},
+  author={Heinlein, Alexander and Rheinbach, Oliver and R{\"o}ver, Friederike and Sandfeld, Stefan and Steinberger, Dominik},
+  year={2019},
+  publisher={Universit{\"a}t zu K{\"o}ln}
+}
```

Comment:
Do you have a link to this one?
I found an article here: https://onlinelibrary.wiley.com/doi/full/10.1002/pamm.201900337